internal/cmd/io.go

```diff
@@ -1,4 +1,4 @@
-package hcsoci
+package cmd
```

Comment:
Should this go in the cmd package? Would we prefer an io package? We can move NewNpipeIO and any other implementations of the UpstreamIO interface over with it.

Reply:
I think the io stuff is fine to keep here. If anything I was thinking about moving the diag.go file into a separate package but couldn't decide.

Reply:
Yea that makes sense. If we ever have more implementations of UpstreamIO I think we can take a look again but I think it's fine here for now.

Reply:
I don't think we have much need for an `io` package right now. It seems like `UpstreamIO` is pretty much only used in the context of `cmd`, so it's fine for them to live together.